packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -73,13 +69,12 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
     this._lastValidValue = value;
 
     // Ensure that the autocalculated precision is not negative.
-    const precision = props.precision || Math.max(calculatePrecision(props.step!), 0);
+    this._precision = props.precision || Math.max(calculatePrecision(props.step!), 0);
```

Comment:
Please avoid the use of `!`! Do a type guard or otherwise, add comments in every place you use `!` to justify this usage.

`!` usages spread like a virus through codebases and it soon becomes impossible to reason about the correctness of the code.

Reply:
@aneeshack4 is this resolved? I see the code is changed, but '!' appear to still be present.